Intégration de la version mobile

Sylvaner opened the pull request:
Suppression du menu si la version mobile n'est pas installée
Ligne de commande pour l'installer par apt si l'installation de nextdom s'est fait comme ça

A tester : La sélection de la commande d'installation au clic sur différents navigateur, internet explorer n'a pas pu être testé.

Sylvaner committed:
Intégration de la version mobile

Suppression du menu si la version mobile n'est pas installée
Ligne de commande pour l'installer par apt si l'installation de nextdom s'est fait comme ça

A tester : La commande d'installation sur différents navigateur, internet explorer n'a pas pu être testé.

Gis70 reviewed (approved):
OK, par contre pour valider l'instal via apt il faut un deb, donc c'est à moitié validé :)

Sylvaner commented:
Ou git clone

vinceg77 reviewed (changes requested):
testé sur mon Xiaomi Mi8 android avec :
- Chrome mobile
- Opéra mobile
- Firefox mobile
- Navigateur Mi
Tout fonctionne c'est à dire que ca copie bien la commande qui s'affiche en fonction du type d'install.

Par contre je reste très dubitatif sur la fonctionnalité....

Approuvé à 1/3 donc !

cyrilphoenix71 committed:
Update connexion design

+ 403, 404, 500 page design
+ moving connexion page
Fix tests


change image 500


forget image

vinceg77 reviewed (approved):
Après concertation auprès de ma hiérarchie :) (@GiDom) je valide cette PR parce qu'elle fonctionne (mais toujours à 1/3)
Mais je pense toujours pas qu'il faille l'intégrer dans une prochaine release de Nextdom tant cette fonction ne me parait pas au standard ! Le prenez pas mal.
